### DISCUSSION TITLE
Developer's setup

### DISCUSSION THREAD

**Discussion by Klurige** [F43nd1r/Acrarium#436] [Q&A]
Is there a guide somewhere about how to work with the source?

**Comment by F43nd1r** [comment]
No guide. It's just a normal gradle project for a spring boot app. Do you have specific questions about the setup?

**Reply by Klurige** [reply]
I may fall short due to my inexperience with springboot, but I tried just running the app and get an error that says "Failed to determine suitable jdbc url". So that would be my first issue. But in general I would like to know how to handle configuration files and also perhaps testruns. Do always test it in a docker container?

**Reply by F43nd1r** [reply]
Ah I guess that's not documented. The application needs a database configured to run locally, same as the published version. You can test in or out of docker containers.
